src/ophys_etl/modules/roi_cell_classifier/compute_classifier_artifacts.py

```diff
@@ -102,15 +111,22 @@ def run(self):
         corr_img = normalize_array(array=corr_img,
                                    lower_cutoff=q0,
                                    upper_cutoff=q1)
-
         self.logger.info("Normalized images...")
 
+        selected_rois = self.args['selected_rois']
+        if selected_rois is None:
+            selected_rois = []
```

Comment:
A slight simplification here is if `selected_rois` is not passed then you can set `selected_rois` to be 
```
selected_rois = [roi['id'] for roi in extract_roi_list
```

then you don't need line 127

Reply:
Changed the order a bit and no the `None` test causes the loop you suggest above.